### PR TITLE
Add aussie_broadband documentation

### PR DESCRIPTION
**Pull request by nickw444**
## Proposed change

Adds documentation for the proposed `aussie_broadband` integration in https://github.com/home-assistant/core/pull/53552

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/53552
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2750
- ~This PR fixes or closes issue:~ 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/aussie_broadband.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ha_config_flow: true
+ha_codeowners:
+- '@nickw444'
+ha_domain: aussie_broadband
+ha_platforms:
+- sensor
+- camera
+---
+
+The `aussie_broadband` integration uses the [Aussie Broadband API](https://myaussie-api.aussiebroadband.com.au) as a source for internet usage data.
```
Comment (frenck) [inline source/_integrations/aussie_broadband.markdown]:
```suggestion
The Aussie Broadband integration uses the [Aussie Broadband API](https://myaussie-api.aussiebroadband.com.au) as a source for internet usage data.
```

File: source/_integrations/aussie_broadband.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Setup
+
```
Comment (frenck) [inline source/_integrations/aussie_broadband.markdown]:
```suggestion
```

File: source/_integrations/aussie_broadband.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Aussie Broadband
+description: Instructions on how to integrate Aussie Broadband within Home Assistant.
+ha_category:
+- Network
```
Comment (frenck) [inline source/_integrations/aussie_broadband.markdown:5]: The category & platforms seems a bit off? Should it also have a camera category?

File: source/_integrations/aussie_broadband.markdown [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ha_release: 2021.9
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+- '@nickw444'
+ha_domain: aussie_broadband
+ha_platforms:
+- sensor
+- camera
```
Comment (frenck) [inline source/_integrations/aussie_broadband.markdown]: A new integration can only bring in only a single platform, so this seems to be incorrect?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/aussie_broadband.markdown (hunk 4 above)
Comment (nickw444) [reply source/_integrations/aussie_broadband.markdown]: Fair call, I thought the additional platform (camera in this case) was trivial enough, but I'll follow your guidance and strip back the PR in core to be only for `sensor`.

File: source/_integrations/aussie_broadband.markdown (hunk 3 above)
Comment (nickw444) [reply source/_integrations/aussie_broadband.markdown:5]: I wasn't sure what the possible "categories" were in this case, so just was guessing based on what I saw in other pages. Should this be `Network`, `Sensor`, and `Camera` (when `camera` platform is added in a subsequent PR)?

File: source/_integrations/aussie_broadband.markdown (hunk 4 above)
Comment (nickw444) [reply source/_integrations/aussie_broadband.markdown]:
```suggestion
- sensor
```

File: source/_integrations/aussie_broadband.markdown (hunk 3 above)
Comment (nickw444) [reply source/_integrations/aussie_broadband.markdown:5]: I've added `Sensor` in the interim 😄 Will await your advice on this though

**Comment by bdraco** [comment]
@Bre77 any more adjustments here ?

**Comment by Bre77** [comment]
> @Bre77 any more adjustments here ?

No, I believe we are good to go.

File: source/_integrations/aussie_broadband.markdown [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+title: Aussie Broadband
+description: Instructions on how to integrate Aussie Broadband within Home Assistant.
+ha_category:
+- Network
+- Sensor
+ha_release: 2021.10
```
Comment (bdraco) [inline source/_integrations/aussie_broadband.markdown]: This needs to be adjusted

**Review by bdraco — CHANGES_REQUESTED** [review]

File: source/_integrations/aussie_broadband.markdown (hunk 5 above)
Comment (Bre77) [inline source/_integrations/aussie_broadband.markdown]:
```suggestion
ha_release: 2022.01
```
@nickw444 can you push this through?

Comment (Bre77) [reply source/_integrations/aussie_broadband.markdown]: Is `2022.01` appropriate?

**Review by Bre77 — CHANGES_REQUESTED** [review]

File: source/_integrations/aussie_broadband.markdown [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: Aussie Broadband
+description: Instructions on how to integrate Aussie Broadband within Home Assistant.
+ha_category:
+- Network
+- Sensor
+ha_release: 2022.01
```
Comment (bdraco) [inline source/_integrations/aussie_broadband.markdown]: 2022.02

We skipped the 01 release

**Comment by bdraco** [comment]
Likely needs a rebase as well

File: source/_integrations/aussie_broadband.markdown (hunk 6 above)
Comment (Bre77) [inline source/_integrations/aussie_broadband.markdown]:
```suggestion
ha_release: 2022.2
```
Sorry @nickw444 they skipped 1. I also raised a PR to rebase your branch at https://github.com/nickw444/home-assistant.github.io/pull/2

**Review by Bre77 — CHANGES_REQUESTED** [review]

File: source/_integrations/aussie_broadband.markdown (hunk 6 above)
Comment (frenck) [reply source/_integrations/aussie_broadband.markdown]: Please note, we don't use a leading zero.

`2022.2`

**Review by bdraco — APPROVED** [review]

File: source/_integrations/aussie_broadband.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/aussie_broadband.markdown:5]: Lets leave it at this.

**Review by frenck — APPROVED** [review]
Thanks, @nickw444 👍

**Pull request merged**